country_plan/models.py

```diff
@@ -0,0 +1,135 @@
+    requested_amount = models.FloatField(verbose_name=_('Requested Amount'), blank=True, null=True)
+    people_targeted = models.IntegerField(verbose_name=_('People Targeted'), blank=True, null=True)
+    is_publish = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f'{self.country}'
+
+    def full_country_plan_mc(self):
+        membership_coordinations_by_sector = {
+            mc.sector: mc
+            for mc in self.country_plan_mc.all()
+        }
+        return [
+            membership_coordinations_by_sector.get(sector) or MembershipCoordination(sector=sector)
+            for sector, _ in MembershipCoordination.Sector.choices
+        ]
+
+
+class StrategicPriority(models.Model):
+    class Type(models.TextChoices):
+        CLIMATE_AND_ENVIRONMENTAL_CRISIS = 'climate_and_environmental_crisis', _('Climate and environmental crisis')
+        EVOLVING_CRISIS_AND_DISASTERS = 'evolving_crisis_and_disasters', _('Evolving crisis and disasters')
+        GROWING_GAPS_IN_HEALTH_AND_WELLBEING = 'growing_gaps_in_health_and_wellbeing', _('Growing gaps in health and wellbeing')
+        MIGRATION_AND_IDENTITY = 'migration_and_identity', _('Migration and Identity')
+        VALUE_POWER_AND_INCLUSION = 'value_power_and_inclusion', _('Value power and inclusion')
+
+    country_plan = models.ForeignKey(
+        CountryPlan, on_delete=models.CASCADE,
+        verbose_name=_('Country Plan'),
+        related_name='country_plan_sp',
+    )
+    type = models.CharField(max_length=100, choices=Type.choices, verbose_name=_('Type'))
+    funding_requirement = models.FloatField(verbose_name=_('Funding Requirement'), blank=True, null=True)
+    people_targeted = models.IntegerField(verbose_name=_('People Targeted'), blank=True, null=True)
+
+    class Meta:
+        unique_together = ('country_plan', 'type')
+
+    def __str__(self):
+        return f'{self.type}'
+
+
+class MembershipCoordination(models.Model):
+    class Sector(models.TextChoices):
+        CLIMATE = 'climate', _('Climate')
+        CRISIS = 'crisis', _('Crisis')
+        HEALTH = 'health', _('Health')
+        MIGRATION = 'migration', _('Migration')
+        INCLUSION = 'inclusion', _('Inclusion')
+        ENGAGED = 'engaged', _('Engaged')
+        ACCOUNTABLE = 'accountable', _('Accountable')
+        TRUSTED = 'trusted', _('Trusted')
+
+    country_plan = models.ForeignKey(
+        CountryPlan,
+        on_delete=models.CASCADE,
+        verbose_name=_('Country Plan'),
+        related_name='country_plan_mc',
+    )
+    sector = models.CharField(max_length=100, choices=Sector.choices, verbose_name=_('Sector'))
+    national_society = models.ForeignKey(Country, on_delete=models.CASCADE, related_name='national_society_mc')
+    has_coordination = models.BooleanField(default=False)
+
+    class Meta:
+        unique_together = ('country_plan', 'national_society', 'sector')
+
+    def __str__(self):
+        return f'{self.national_society.iso3}-{self.type}'
```

Comment:
It seems to be unresolved attribute (self.type).
![kép](https://user-images.githubusercontent.com/6050643/203931163-3dcb7068-c108-493a-be15-97953e255958.png)